### ISSUE TITLE
Treat display_type e-journal the same as journal

### ISSUE THREAD

**Issue by chrpr** [NYULibraries/exlibris-nyu#32]
Hi @barnabyalter,

Per Slack conversation, I'm reporting the `e-journal` vs `journal` issue.

Journal's in Umlaut should have summary instead of item level holdings. Correctly working example:
https://getit.library.nyu.edu/go/9389281

Primo admins implemented e-journal as a type, and these are functioning incorrectly, using the book logic: https://getit.library.nyu.edu/go/9389269

We think the issue is due to this line, matching journal but not e-journal.

I recommend changing to something like:
 `(display_type && (display_type.upcase == 'JOURNAL' || display_type.upcase == 'E-JOURNAL' ))`

Let me know if you have any questions.

Thanks!

**Comment by barnabyalter** [comment]
@chrpr Do you have any examples of this NOT functioning correctly with a primo record that is no a dedupmrgr? Tx

**Comment by chrpr** [comment]
@barnabyalter No, I don't. I'm not sure it's possible currently for this to happen for non dedup.

I don't think e-journal is currently mapped as a display_type for aleph data. Only in NYUMARCIT & NSMARCIT use it. And when these MARCIT sources don't dedup, there's no Aleph holdings there to have their logic broken.

@mth0611 Does that make sense to you? Can you think of a case where there'd be non-dedup examples of this?

**Comment by barnabyalter** [comment]
@chrpr What I'm trying to do is write a failing test but I don't think I properly understand what a correct record is supposed to look like. Could you describe what fields in the incorrect record are incorrect and what they should be?

**Comment by chrpr** [comment]
@barnabyalter Locations shouldn't be repeated under "Copies in Library".

So, for the incorrect record, there are currently 36 "NYU Bobst Main Collection" lines, each with its own "More Info" button. For journals, these are collapsed into a single, much more user friendly, line. This collapsing should be true of each location.

Does that help?

**Comment by ajones3066** [comment]
Just a thought- another way to go would be to adjust the openurl template for marcit (instead of relying on display/type, we could hardcode 'journal').  This would mean no code needed to change in umlaut.

A

**Comment by barnabyalter** [comment]
@ajones3066 @chrpr i don't know what that entails but if you've decided to change this somehow on the data end please let me know before I implement this.

**Comment by barnabyalter** [comment]
@chrpr @ajones3066 I deployed this change into dev.getit.library.nyu.edu. Let me know if it fixes the specific issue, but I'll reiterate the point I made above about changing the data as a preference. Tx

**Comment by chrpr** [comment]
@barnabyalter :+1: This looks spot on to me. The examples reported on Oct 15 seem resolved except 2 edge cases that seem to reflect differences in data between prod & dev aleph. I'd call this good to go.

I believe primo admins would rather not diverge from Ex Libris' choices re: underlying data, but I'd recommend you discus that with @mth0611 & @ajones3066 .

My recommendation is :shipit: 

Thanks!
